setup.py

```diff
@@ -12,7 +13,7 @@
       license='BSD-3',
       packages=['xdem'],
       install_requires=['numpy', 'scipy', 'rasterio', 'geopandas', 'pyproj'],
-      extras_require={'rioxarray': ['rioxarray'], 'richdem': ['richdem'], 'pdal':['pdal']},
+      extras_require={'rioxarray': ['rioxarray'], 'richdem': ['richdem'], 'pdal': ['pdal']},
```

Comment:
tqdm is also required to run the tests

Reply:
Not just the tests, actually, but the entire `coreg.py`

Reply:
Added it as a dependency in 8c76cbe

Reply:
Also geoutils obviously is a dependency...

Reply:
Added geoutils as a dependency in 6ebdad9